.github/pages/index.md

```diff
@@ -1,13 +1,11 @@
 ---
-title:  "Editor Documentation"
+title:  "HowTo for Admins"
 nav_exclude: true
 ---
 
-# Editor Documentation
-
 {: .fs-6 }
-Documentation for editors of the Justice UK website. Guides to creating, editing and scheduling content.
+Welcome to the HowTo for Justice UK Admins. Guides to creating, editing and scheduling content.
```

Comment:
The reference to Admin could read: 
- Welcome to the HowTo for the Justice UK Dashboard, or
- Welcome to the HowTo for Justice UK admin pages

Something like that 😄

Reply:
Or something like:
- Welcome to the HowTo Guide for Justice UK admin pages

Maybe?

Reply:
Cheers! I'll go with: 

> Welcome to the HowTo Guide for Justice UK admin pages